region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -1314,6 +1314,242 @@
       ],
       "devNote": "TODO: Sparking diagonally through the door cannot be shown as an alternative."
     },
+    {
+      "link": [2, 3],
+      "name": "Cross Room Blue Speed Jump with Hi-Jump",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 0,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$2.5",
+          "maxExtraRunSpeed": "$2.F"
+        }
+      },
+      "requires": [
+        "HiJump",
+        "canLateralMidAirMorph",
+        "canTrickyJump",
+        {"heatFrames": 150}
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "note": [
+        "Jump into the room with blue speed.",
+        "Morph and unmorph to clear Samus' horizontal momentum while retaining blue,",
+        "to break the right-most bomb blocks above and reach the top of the room."
+      ],
+      "devNote": [
+        "Higher run speeds can also work, but with greater difficulty."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Cross Room Blue Speed Jump",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 0,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$6.0",
+          "maxExtraRunSpeed": "$6.1"
+        }
+      },
+      "requires": [
+        "canTrickyDashJump",
+        "canInsaneJump",
+        "canLateralMidAirMorph",
+        {"heatFrames": 120}
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "note": [
+        "Jump into the room with blue speed and a specific amount of horizontal momentum, morphing through the transition.",
+        "Unmorph to clear Samus' horizontal momentum while retaining blue,",
+        "to break the right-most bomb blocks above and reach the top of the room."
+      ],
+      "detailNote": [
+        "It can help to pause buffer the morph:",
+        "time a pause to hit just after Samus jumps and aims down,",
+        "then hold down during the unpause black screen to buffer the morph."
+      ],
+      "devNote": [
+        "Higher run speeds could also work, but with greater difficulty."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Cross Room Very Slow Blue Speed Jump with Hi-Jump and Wall Jump",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 0,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$0.C",
+          "maxExtraRunSpeed": "$0.F"
+        }
+      },
+      "requires": [
+        "HiJump",
```

Comment:
This is probably an insane level shortcharge.

Reply:
Currently a speed of $0.F or lower would be a Beyond shortcharge, as $1.0 is the lowest speed assumed possible in Insane: https://github.com/vg-json-data/sm-json-data/blob/master/strats.md#blue-run-speed-table

Makes sense to reduce the $1.0 in that table down to $0.F, to make this logical for Insane. Updated that now.